sqlite/src/func.c

```diff
@@ -3062,6 +3087,7 @@ void sqlite3RegisterBuiltinFunctions(void){
     FUNCTION(comdb2_starttime,      0, 0, 0, comdb2StartTimeFunc),
     FUNCTION(comdb2_user,           0, 0, 0, comdb2UserFunc),
     FUNCTION(comdb2_last_cost,      0, 0, 0, comdb2LastCostFunc),
+    FUNCTION(comdb2_snapshot_lsn,   0, 0, 0, comdb2SnapshotLsnFunc),
     FUNCTION(checksum_md5,          1, 0, 0, md5Func),
     FUNCTION(compress,              1, 0, 0, compressFunc),
     FUNCTION(uncompress,            1, 0, 0, uncompressFunc),
```

Comment:
The code here is really simple so I think it's okay to not have a test in the suite for it but I think we should at least do some ad hoc testing of the different cases before merging.